### PR TITLE
Fix: adds datacite tests for Zenodo records, moves _detag function fr…

### PR DESCRIPTION
**Pull request by seasidesparrow**
…om JATSParser to BaseBeautifulSoupParser and renames tagsets from JATS_ to HTML_

 	modified:   adsingestp/parsers/base.py
 	modified:   adsingestp/parsers/datacite.py
 	modified:   adsingestp/parsers/jats.py
 	new file:   tests/stubdata/input/zenodo_test.xml
 	new file:   tests/stubdata/input/zenodo_test2.xml
 	new file:   tests/stubdata/input/zenodo_test3.xml
 	new file:   tests/stubdata/input/zenodo_test4.xml
 	modified:   tests/stubdata/output/datacite_schema3.1_example-full.json
 	modified:   tests/stubdata/output/datacite_schema4.1_example-full.json
 	modified:   tests/stubdata/output/datacite_schema4.1_example-software.json
 	modified:   tests/stubdata/output/datacite_schema4_example-habanero-pdsdataset.json
 	new file:   tests/stubdata/output/zenodo_test.json
 	new file:   tests/stubdata/output/zenodo_test2.json
 	new file:   tests/stubdata/output/zenodo_test3.json
 	new file:   tests/stubdata/output/zenodo_test4.json
 	modified:   tests/test_datacite.py

### CODE REVIEW

**Comment by seasidesparrow** [comment]
Also adds OA/license capture to to Datacite parser

File: adsingestp/parsers/base.py [hunk 1]
```diff
@@ -471,6 +471,35 @@ class BaseBeautifulSoupParser(IngestBase):
     out of the input XML stream.
     """
 
+    fix_ampersand = re.compile(r"(&amp;)(.*?)(;)")
+
```
Comment (ehenneken) [inline adsingestp/parsers/base.py:475]: I don't know if you have ever encountered this in this context: I have seen cases where ampersands got encoded as `__amp__amp;`

Comment (seasidesparrow) [reply adsingestp/parsers/base.py:475]: Do you remember which publisher(s) specifically?  I'm looking for an example to make a unit test with.

Comment (ehenneken) [reply adsingestp/parsers/base.py:475]: This file has an example: `/proj/ads/references/sources/MNRAS/0423/iss4.wiley2.xml`

**Review by ehenneken — APPROVED** [review]

**Pull request merged**